web/spacewalk-web.changes

```diff
@@ -1,3 +1,5 @@
+- Target specific div in Select component instead of document.body
```

Comment:
I think this changelog entry could be a bit improved. The changelog is intended for customers, not technical people, and for them things like `div` and `document.body` won't necessarily say anything useful. I'm not even sure if I would add a changelog entry here, since from the customer's point of view, nothing visibly tangible has changed. We've improved the technical situation, yes, but they don't see it verbatim. Alternatively, could write something along the lines of a-la `Improved dropdown layout handling` or something similar which is understandable without a technical background.

Reply:
Fully agree with @Etheryte

Reply:
Yes, you are right. I hadn't really stopped to think about it properly and I think this entry should be removed as you suggested.